stub/src/main/java/io/grpc/stub/ServerCalls.java

```diff
@@ -211,24 +215,26 @@ public void onReady() {
    * @param method an adaptor to the actual method on the service implementation.
    */
   private static <ReqT, RespT> ServerCallHandler<ReqT, RespT> asyncUnaryRequestCall(
```

Comment:
I feel this method is just adding an annoying redirection. Why not just call `new UnaryServerCallHandler<>(method, serverStreaming)` directly whenever `asyncUnaryRequestCall` is called?

Reply:
That sounds fine. Do you mind if I do that as a follow-up?

Reply:
Actually, let me do it as a pre-refactor, since I was going to let this PR "sit" for us to discuss it in the API meeting.